pypeit/par/pypeitpar.py

```diff
@@ -404,7 +404,7 @@ class FlatFieldPar(ParSet):
     For a table with the current keywords, defaults, and descriptions,
     see :ref:`pypeitpar`.
     """
-    def __init__(self, frame=None, illumflatten=None, method=None): #, params=None, twodpca=None):
+    def __init__(self, frame=None, illumflatten=None, tweak_slits = None, method=None): #, params=None, twodpca=None):
```

Comment:
Sorry that I'm nit-picking:  In general, please separate arguments of a function with a comma and a space, and don't include spaces between a keyword argument and its default value.  I.e. do this:

```
def foo(arg1, arg2, kwarg1=0., kwarg2=1.)
```

not what you tend to do, which is:

```
def foo(arg1,arg2,kwarg1 = 0.,kwarg2 = 1.)
```

Reply:
Fixed. Thanks!